src/shrub/base.py

```diff
@@ -9,9 +9,7 @@
 NAME_KEY = "name"
 
 
-class EvergreenBuilder:
-    __metaclass__ = abc.ABCMeta
-
+class EvergreenBuilder(abc.ABC):
```

Comment:
The changes in this file are to fix a deprecation warning with the old code. It is the only change in the original shrub code.